lib/Server.js

```diff
@@ -489,7 +485,7 @@ Server.prototype.listen = function(port, hostname) {
 	});
 	sockServer.on("connection", (conn) => {
 		if(!conn) return;
-		if(!this.checkHost(conn.headers)) {
+		if(!this.checkHost(conn.hostname)) {
```

Comment:
it looks obvious as to why this was changed to `hostname` but what I'm wondering is if there was reason it was using `headers`. I think we may need tests around this to be certain.